ui/Screen/Org/Members.svelte

```diff
@@ -6,10 +6,22 @@
  LICENSE file.
 -->
 <script lang="typescript">
-  import { Avatar, List, StyledCopyable } from "ui/DesignSystem";
   import type * as org from "ui/src/org";
+  import type * as project from "ui/src/project";
+  import * as userProfile from "ui/src/userProfile";
+  import * as Session from "ui/src/session";
+
+  import { Avatar, List, StyledCopyable } from "ui/DesignSystem";
 
   export let members: org.Member[];
+  const session = Session.getUnsealedFromContext();
+
+  function onSelect({ detail: member }: { detail: project.User }) {
```

Comment:
Just a small suggestion: I find it easier to follow if functions are named by what they do and not by what they are called. I’d suggest calling this `openUserProfile` and having `on:select={openUserProfile}`.

Reply:
a01379e1474e3189bb7c25994ab2ce5803affedd